sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -150,6 +150,9 @@ export interface InternalPipelineOptions extends PipelineOptions {
     loggingOptions?: LogPolicyOptions;
 }
 
+// @public
+export function isRestError(e: unknown): e is RestError;
```

Comment:
Did we want to make this part of the public API? I know we were considering having this live in core-util so maybe we should hold off exposing it?

Reply:
I think the typeguard for this should live here because it's where RestError lives, I think when we define custom errors, we should always export a typeguard as well (and consuming SDKs can re-export both the error and the typeguard) - otherwise we'd maybe create a dependency from core-util onto core-rest-pipeline for the definition, which would make a bad cycle. I really don't want core-util to depend on *anything* except types

Reply:
Ah I see, that makes sense - thanks!